e2e_tests/iris_classifier_example.py

```diff
@@ -3,7 +3,7 @@
 from bentoml.handlers import DataframeHandler
 
 
-@env(pip_dependencies=['scikit-learn'])
+@env(pip_dependencies=['scikit-learn==0.22'])
```

Comment:
just use `auto_pip_dependencies`, make it consistent with the quickstart guide example?